app/views/admin/domains/_table.pug

```diff
@@ -26,7 +26,14 @@ include ../../_pagination
             td.align-middle.text-center
               = t(titleize(humanize(domain.plan)))
             td.align-middle.text-center
-              = domain.max_recipients_per_alias
+              form.ajax-form.confirm-prompt.d-inline-block(action=l(`/admin/domains/${domain.id}`), method="POST", autocomplete="off")
+                input(type="hidden", name="_csrf", value=ctx.csrf)
+                input(type="hidden", name="_method", value="PUT")
+                .input-group
+                  select#input-max-recipients-per-alias(name='max_recipients_per_alias').form-select.form-select-sm
+                    each option in OPTIONS_MAX_RECIPIENTS
+                      option(value=option selected=option === domain.max_recipients_per_alias)= option
```

Comment:
You're missing a comma in between `value=option selected=option` (our code style).  Not sure if pug linter would have caught this, but I'd fix this and possibly skim through anywhere else you might forget it.

Reply:
Also there could be an arbitrary value here in the database so it should be an `input(type="number", inputmode="numeric", step=10, ..., value=domain.max_recipients_per_alias)`

Reply:
oh i forgot about `step=`.... that is much better.